Example-CustomUserGroups-REXX/USRGRP01.rexx

```diff
@@ -0,0 +1,3123 @@
+
+  Do memIdx = 1 to numOfCustUsrGrpMems
+
+    /*
+     * Grab the next entry within the Custom Groups List
+     */
+    nextMemberEntry = GetJSONArrayEntry(groupsList, memIdx - 1)
+
+    customUsrGrpMemberName = FindJSONValue(,
+      nextMemberEntry,,
+      GROUP_MEMBER_NAME_KEY,,
+      HWTJ_STRING_TYPE,
+    )
+
+    If WasJSONValueFound(customUsrGrpMemberName) = FALSE Then Do
+      Say 'Failed to find Custom User Group Member name'
+      Return 0
+    End
+
+    customUsrGrpMemberURI = FindJSONValue(,
+      nextMemberEntry,,
+      GROUP_MEMBER_URI_KEY,,
+      HWTJ_STRING_TYPE,
+    )
+
+    If WasJSONValueFound(customUsrGrpMemberURI) = FALSE Then Do
+      Say 'Failed to find Custom User Group Member URI'
+      Return 0
+    End
+
+    customUserGroupMems.customUsrGrpMemberName = customUsrGrpMemberURI
+
+    Say 'Custom User Group Member Name: ' ||,
+        customUsrGrpMemberName ||,
+        ' | URI: ' ||,
+        customUserGroupMems.customUsrGrpMemberName
+  End
+
+  customUserGroupMems.0 = numOfCustUsrGrpMems
+
+Return 0
+
+/**********************************************************************
+ * Function: AddNewGroupMember
+ *
+ * Purpose: Add a new LPAR group member to the specified Custom User
+ *          Group
+ *
+ * Input: GROUP_ID - The ID of the group to target
+ *        LPAR_URI - The URI of the lpar object to add to the group
+ *        TARGET_CPC_NAME - The name of the CPC to target
+ *
+ * Output: None
+ *
+ * Side-Effects:
+ *
+ *  1. Adds an lpar object to the custom user group
+ *
+ **********************************************************************/
+AddNewGroupMember:
+
+  GROUP_ID = Arg(1)
+  LPAR_URI = Arg(2)
+  TARGET_CPC_NAME = Arg(3)
+
+  ADD_GROUP_MEMBER_URI = GROUP_ID || '/operations/add-member'
+
+  ADD_GROUP_MEMBER_BODY = '{"object-uri":"' || LPAR_URI || '"}'
+
+  postRequestResponse = PostRequest(,
+    ADD_GROUP_MEMBER_URI,,
+    ADD_GROUP_MEMBER_BODY,,
+    TARGET_CPC_NAME,
+  )
+
+Return 0
+
+/**********************************************************************
+ * Function: RemoveGroupMember
+ *
+ * Purpose: Remove a LPAR group member from the specified Custom User
+ *          Group
+ *
+ * Input: GROUP_ID - The ID of the group to target
+ *        LPAR_URI - The URI of the lpar object to add to the group
+ *        TARGET_CPC_NAME - The name of the CPC to target
+ *
+ * Output: None
+ *
+ * Side-Effects:
+ *
+ *  1. Removes an lpar object from the custom user group
+ *
+ **********************************************************************/
+RemoveGroupMember:
+
+  GROUP_ID = Arg(1)
+  LPAR_URI = Arg(2)
+  TARGET_CPC_NAME = Arg(3)
+
+  ADD_GROUP_MEMBER_URI = GROUP_ID || '/operations/remove-member'
+
+  ADD_GROUP_MEMBER_BODY = '{"object-uri":"' || LPAR_URI || '"}'
+
+  postRequestResponse = PostRequest(,
+    ADD_GROUP_MEMBER_URI,,
+    ADD_GROUP_MEMBER_BODY,,
+    TARGET_CPC_NAME,,
+    '',,
+    '',,
+    '',
+  )

```

Comment:
i almost rather see you create another flavor of a PostRequest method so you don't have to pass in the empty string parms